src/styles/blocks/content.scss

```diff
@@ -0,0 +1,21 @@
+.content {
+  display: flex;
+  align-items: center;
+}
+
+.content__main {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 50px;
+
+  &__card {
+    box-sizing: border-box;
+    width: 200px;
+    margin-right: 50px;
+  }
+
+  &--row-4 {
+    width: 950px;
+  }
+}
```

Comment:
I don't see this class anywhere in the index.html and you have catalog for the same purpose